### ISSUE TITLE
SoundSystemWrapper - performance issue.

### ISSUE THREAD

**Issue by Gliby** [GameModsBR/VoiceChat#13]
https://github.com/GameModsBR/VoiceChat/blob/master/src/main/java/net/gliby/voicechat/client/sound/SoundSystemWrapper.java
The class is a huge hack of Minecraft's inner sound system, most likely is diminishing performance.  The solution would be to use Forge's AccessTransformers, in fact it was used in the newer version 0.6.
1.7.10 gvc_at.cfg file https://gist.github.com/Gliby/c02b2216186312448ae7f4f762a075cf

**Comment by utkabobr** [comment]
http://www.minecraftforge.net/forum/topic/2122-accesstransformers/
Can it be useful?

**Comment by Gliby** [comment]
No. Use this https://tutorials.darkhax.net/tutorials/access_transformers/
